src/oidc/tokenRefresher.ts

```diff
@@ -16,20 +16,28 @@ limitations under the License.
 
 import { IdTokenClaims, OidcClient, WebStorageStateStore } from "oidc-client-ts";
 
-import { TokenRefreshFunction } from "../http-api";
+import { AccessTokens } from "../http-api";
 import { IDelegatedAuthConfig } from "../client";
 import { generateScope } from "./authorize";
 import { discoverAndValidateAuthenticationConfig } from "./discovery";
 import { logger } from "../logger";
 
 /**
  * @experimental
- * Class responsible to refreshing OIDC access tokens
+ * Class responsible for refreshing OIDC access tokens
+ *
+ * Client implementations will likely want to override {@link persistTokens} to persist tokens after successful refresh
+ *
  */
-export abstract class OidcTokenRefresher {
+export class OidcTokenRefresher {
+    /**
+     * Used to determine the OidcClient has been initialised
+     * and is ready to start refreshing tokens
+     * throws when client initialisation fails
```

Comment:
[Long sentence](https://pr3769--js-sdk-docs-previews.netlify.app/stable/classes/oidctokenrefresher#oidcClientReady) is long and not very clear. "Throws when..." should be a separate sentence.

Also, this is odd wording. It isn't really up to an API to define how that is used. You could say "*Can* be used to determine the OidcClient has been initialised", but I think it would be simpler just to say:

```suggestion
     * Promise which will complete once the OidcClient has been initialised
     * and is ready to start refreshing tokens.
     *
     * Will reject if the client initialisation fails.
```